src/ports/nova/templates/README.md

```diff
@@ -0,0 +1,30 @@
+{{{ basic-heading }}}
+
+# Installation
+
+1. Open Nova Extension Library from Nova Extensions menu
```

Comment:
Are the steps to install right? The compiled file in the packages folder is only the hello-aura.json that has only one color. In the src/ports folder template folder should have files that will be transformed by the template engine, and extra folder to files and folder that will be cloned as it is. Please, take a look at [how I did it in the vscode port](https://github.com/daltonmenezes/aura-theme/tree/main/src/ports/vscode)

If you need the final `AuraTheme.novaextension` be a zip file, take a look [how I did it in the Telegram port](https://github.com/daltonmenezes/aura-theme/blob/main/src/ports/telegram/index.ts)

Reply:
I as following the instructions for creating a port which is why the aura.json file is in there.  That file isn’t actually needed for the extension.  Nova has it’s own extension store that I was going to submit to once you approved the port. AuraTheme.novaextension just needs to be submitted to the Nova extensions and then users can access it from within the App, kind of live VS code.
![CleanShot 2022-04-19 at 07 49 43](https://user-images.githubusercontent.com/9391497/164008677-767b660f-44b7-40d5-a842-ad7563665cea.png)

Reply:
@DanKlaver15 In general, it's important that no theme depends on the store (even if it will be published later), we need to make use of the Aura generator in case any color or color tone changes the change is done in bulk without effort, this is only possible if we follow the creation based on the generator that will translate the tokens. Take a look at how [vscode port](https://github.com/daltonmenezes/aura-theme/tree/main/src/ports/vscode) was made, please.

As for in-store publishing, this is a dilemma for me. I currently prefer official themes to be put in stores by me, if you don't mind. Opening official publications in stores can generate some problems, such as, for example, an update in Aura depends on the author of the port, or the author of the port decides not to follow the Aura guidelines. Of course, unofficial ones can be done without problems, but in this case it will not depend on our support.

About that red line in the tab the [Buttons section in DOC](https://docs.nova.app/extensions/themes/#buttons) says:

`For any of these selectors, adding the document identifier specifies that the appearance should be applied to buttons appearing in document tabs, such as meta.document.button.`

Maybe this is the way to change that red color, can you take a look at this, please?

I noticed that you only used one variant which is Aura.dark, is it not possible to use the available variants as was done in the vscode port?

The other change requests I made are still valid. Thanks. 💜